### ISSUE TITLE
Displaying more info on color picker

### ISSUE THREAD

**Issue by rwwagner90** [mockingbot/electron-color-picker#8]
Would it be possible to show the hex or RGB or maybe a color name or swatch on the picker? Something like this:

<img width="484" alt="Screen Shot 2019-12-15 at 2 12 35 PM" src="https://user-images.githubusercontent.com/2640861/70867595-1db7a300-1f45-11ea-8c83-7bca38efdde2.png">

**Comment by ThatBean** [comment]
The binary comes from: [mockingbot/mb_colorpicker_desktop_native](https://github.com/mockingbot/mb_colorpicker_desktop_native)
Related: https://github.com/mockingbot/electron-color-picker/issues/5

**Comment by RobbieTheWagner** [comment]
@ThatBean so are you saying we should open an issue there?

**Comment by ThatBean** [comment]
Yes, the binary source is there, and this repo is for providing an Electron npm package

Als stating `[Feature]` in the issue title will better help us track them

**Comment by RobbieTheWagner** [comment]
@ThatBean would you be able to transfer this issue to that repo please?
